src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -517,15 +518,14 @@ private void createMainTable() {
                 entryTypesManager,
                 taskExecutor,
                 fileUpdateMonitor);
-
+        // conntent binding does not work as it won't trigger the needsEntries checker
```

Comment:
I did not quite get the reference to `needsEntries checker`. Thus, I tried to find it:

```suggestion
        // content binding does not work as it does not trigger the ManageKeywordsAction#needsEntriesSelected checker
```

Reply:
I added a clarification, ActionHelper#needsEntriesSelected - enables or disables the menu items